torch/linalg/__init__.py

```diff
@@ -400,6 +400,53 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+solve = _add_docstr(_linalg.linalg_solve, r"""
+linalg.solve(input, other, *, out=None) -> Tensor
+
+Computes the solution ``x`` to the matrix equation ``matmul(input, x) = other``
+with a square matrix, or batches of such matrices, :attr:`input` and multiple right-hand side vectors :attr:`other`.
```

Comment:
"and multiple right-hand side"...

Would it be more accurate to say "and one or more..."?